scripts/qesap/test/conftest.py

```diff
@@ -320,6 +320,8 @@ def mock_call_ansibleplaybook():
     ```
     '''
     def _callback(playbook_cmd):
-        return mock.call(cmd=playbook_cmd, env={'ANSIBLE_PIPELINING': 'True'})
+        original_env = dict(os.environ)
```

Comment:
This doesn't create a copy of the `os.environ` dictionary.  

```suggestion
        original_env = os.environ.copy()
```